### ISSUE TITLE
[questions/qa-floating-times] textContent won't change

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-drafts#109]
[source] (https://w3c.github.io/i18n-drafts/questions/qa-floating-times.en) [en]

> `<p>Publication Date: <time datetime="2017-04-30">April 30, 2017</time></p>`
> 
> This seems unambiguous. Yet for some reason when Martina, who is in Buenos Aires, Argentina, opens the Web site, she sees "Saturday edition" and "29 April 2017" as the date for her copy.

Actually, i think she'll see exactly the same thing, because the browser won't change the textContent of the time element afaia.  It's only if the date is being generated that you'd see it produce something differently, surely?

Also, i think there's an unspoken assumption that the newspaper was published in the GMT zone, although in April DST would apply. Would be best to clarify the location of the original date stamp, and maybe choose another date that is directly tied to UTC?

**Comment by r12a** [comment]
The first part of this may be the same comment as https://github.com/w3c/i18n-drafts/issues/99

**Comment by aphillips** [comment]
Good point about the text content. I'll make clearer that the <time> element was used to populate some other value.

> Also, i think there's an unspoken assumption that the newspaper was published in the GMT zone, although in April DST would apply. Would be best to clarify the location of the original date stamp, and maybe choose another date that is directly tied to UTC?

No, that's not the assumption at all. The problem is that timestamp (incremental) time values are tied to UTC. When no hours/minutes/seconds are supplied, that part of the date is zeroed out to midnight. This zeroing out can be using the local time zone, a remote time zone, an offset, or UTC. But once done, the time value is "fixed" in UTC terms. Displaying the date value applies the local offset to the timestamp, resulting in display problems. I will try to clarify.
